tests/src/test/scala/cats/collections/DefaultScalaCheckPropertyCheckConfig.scala

```diff
@@ -27,6 +27,7 @@ object DefaultScalaCheckPropertyCheckConfig {
   final val default: Parameters = Parameters.default
     .withMinSuccessfulTests(if (BuildInfo.isJvm) 50 else 5)
     .withMaxDiscardRatio(if (BuildInfo.isJvm) 5 else 50)
+    .withMaxSize(if (BuildInfo.isJvm) 10 else 5)
```

Comment:
I added this to synchronise with [cats](https://github.com/typelevel/cats/blob/771c6c802f59c72dbc1be1898081c9c882ddfeb0/tests/shared/src/test/scala/cats/tests/CatsSuite.scala) - without it the tests take a really long time to run